### PR TITLE
Additional Wallets

### PR DESCRIPTION
**Pull request by avoocado-dev**

### CODE REVIEW

File: src/connectors/index.tsx [hunk 1]
```diff
@@ -23,6 +29,26 @@ export const injected = new InjectedConnector({
   supportedChainIds: [1], // we only support mainnet
 });
 
+// export const bitski = new BitskiConnector({
+//   clientId: BITSKI_KEY,
+//   redirectUri: 'https://localhost:3000/bitski-callback.html',
+// })
```
Comment (just-toby) [inline src/connectors/index.tsx]: what's the plan with bitski? if you don't plan to use the commented code can you remove it?

File: src/connectors/Bitski.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+//         throw new Error(`Unsupported network ID: ${chainId}`)
+//       }
+//     }
+
+//     const provider = this.bitski.getProvider()
+
+//     const pollForOverlayReady = new Promise<void>(resolve => {
+//       const interval = setInterval(() => {
+//         if (provider.overlayReady) {
+//           clearInterval(interval)
```
Comment (just-toby) [inline src/connectors/Bitski.tsx]: same as below

File: src/utils/constants.tsx [hunk 3]
```diff
@@ -68,4 +68,40 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
...
+  AUTHEREUM: {
+    connector: authereum,
+    name: 'Authereum',
+    iconName: 'authereum.png',
+    description: 'Login using Authereum hosted wallet',
+    href: null,
+    color: '#6748FF',
+    mobile: true
+  }, 
+  // BITSKI: {
```
Comment (just-toby) [inline src/utils/constants.tsx]: same

Comment (avoocado-dev) [reply src/utils/constants.tsx]: Up to you. I figured we might circle back to this pretty quickly. I can remove it

**Review by just-toby — APPROVED** [review]

File: src/utils/constants.tsx (hunk 3 above)
Comment (just-toby) [reply src/utils/constants.tsx]: i think we should remove for now - maybe just put a TODO with relevant info to re-implement when the time comes

Comment (avoocado-dev) [reply src/utils/constants.tsx]: fasho

**Pull request merged**